pkgs/applications/editors/neovim/default.nix

```diff
@@ -108,6 +108,7 @@ let
                 $out/bin/nvim
     '' + optionalString withPython ''
       ln -s ${pythonEnv}/bin/python $out/bin/nvim-python
+      ln -s ${pythonEnv}/bin/python $out/bin/python2
```

Comment:
@steveeJ everything that ends up in ``$out/bin`` gets propageted in nix-env profiles and ends up in a path. as far as i see it, current code created ``$out/bin/nvim-python`` and ``$out/bin/nvim-python3`` which should be pretty specific.

i guess proper way would be now to split the python environment out of main neovim derivation and then wrap nvim executables PATH with its $out/bin.